Changelog.md

```diff
@@ -7,6 +7,7 @@ Language Features:
 
 
 Compiler Features:
+ * Code Generator: Transient storage state variables are now supported by the legacy pipeline.
```

Comment:
I think it's important to mention that these are restricted to value types only.

Reply:
Done.